CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## [v3.0.0] - Unreleased
```

Comment:
should this include the "-alpha.1" too?

Reply:
Well, maybe. Right now it shouldn't really matter.